.gitignore

```diff
@@ -197,3 +197,11 @@ lefthook.yml
 
 ## metro server logs
 metro-server-logs.log
+
+## adb logs
+adb_shell_monkey.log
+adb_install.log
+
+## xcrun logs
+xcrun_install.log
+xcrun_launch.log
```

Comment:
Maybe we should just have a `logs` folder that we ignore? That way we won't be cluttering up root of the repo.

Reply:
yeah good idea.

Reply:
Also, not necessarily in this PR, we could move the iOS related logs to `logs` instead of `ios/logs`:
https://github.com/status-im/status-mobile/blob/d77033e3463ff98e7419d57870b4f8bafc093e2b/ios/logs/README.md?plain=1#L7-L10
https://github.com/status-im/status-mobile/blob/d77033e3463ff98e7419d57870b4f8bafc093e2b/fastlane/Fastfile#L126
https://github.com/status-im/status-mobile/blob/d77033e3463ff98e7419d57870b4f8bafc093e2b/ios/StatusIm.xcodeproj/project.pbxproj#L545
https://github.com/status-im/status-mobile/blob/d77033e3463ff98e7419d57870b4f8bafc093e2b/ios/StatusIm.xcodeproj/project.pbxproj#L581
https://github.com/status-im/status-mobile/blob/d77033e3463ff98e7419d57870b4f8bafc093e2b/ci/Jenkinsfile.ios#L68

Reply:
hmm I'm eager to merge this PR, I'll take this feedback as a separate PR.
Thanks for the feedback!

Reply:
captured here -> https://github.com/status-im/status-mobile/issues/18915